java-squid/src/main/java/org/sonar/java/se/ExplodedGraphWalker.java

```diff
@@ -287,6 +289,15 @@ private void visit(Tree tree, Tree terminator) {
           programState = ProgramState.put(programState, variableTree.symbol(), unstack.b.get(0));
         }
         break;
+      case TYPE_CAST:
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce this switch case number of lines from 8 to at most 5, for example by extracting code into methods. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1151)